src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,20 +77,30 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
-      // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired')) {
+        // Do resume polling since polling is already stop and session is invalid
```

Comment:
if we are not doing an explicit `startPolling` then does it still resume polling? It's confusing that we return null and still say `resume polling`

Reply:
type missing `NOT` 

```
// Do NOT resume polling since session is invalid and polling is already stop
```